postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -159,41 +183,53 @@ def _dbtags(self, db, *extra_tags):
             t.extend(self._tags_no_db)
         return t
 
-    def _get_new_pg_stat_activity(self):
+    def _get_active_connections(self):
+        active_connections = []
+        try:
+            active_connections = self._run_active_conn_query()
+        except Exception as e:
```

Comment:
Sorry to go back on what I said earlier, but I don't think we need this try/catch wrapper after all. I just remembered we have common error handling & reporting in the [main async job loop](https://github.com/DataDog/integrations-core/blob/fac0bffb3d2d3720c4c24a147fd94670953137b9/datadog_checks_base/datadog_checks/base/utils/db/utils.py#L240-L262). That should be enough for all of the `pg_stat_activity` queries we do in `_collect_statement_samples` so we don't need to wrap each of them individually. 

The explains have their own try/catch because we don't want a single query to cause the rest to fail and we should keep those like that.

Reply:
OK, I agree. I went back and forth on this while I was writing this code last night